### ISSUE TITLE
[SCAC1012] Add additional ARIA accessability

### ISSUE THREAD

**Issue by ItsPugle** [ItsPugle/scac#4]
> Migrated from Freedcamp; [SCAC1012](https://freedcamp.com/Spotify_Projects_EQk/SAC_OC6/bugtracker/view/98182).

ARIA tags allow anyone that uses screen readers to have more clear dictation of the on-screen content. As a tool that's designed for everyone, this should be done.
  - [ARIA Overview on W3C](https://www.w3.org/WAI/standards-guidelines/aria/)
  - [ARIA accessibility on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
